specs/schemes/exact/scheme_exact_algo.md

```diff
@@ -0,0 +1,222 @@
+
+## Payment Requirements asset definition 
+
+In the `exact` scheme on Algorand, the `paymentRequirements.asset` **MUST** be a string representing an `Algorand Standard Asset (ASA)` ID unsigned integer instead of an `ERC20` contract address. This **MUST** be validated by the resource server to ensure the `asset` field is valid when using the Algorand scheme.
+
+
+## `X-Payment` header payload
+
+The `payload` field of the `X-PAYMENT` header **MUST** contain `paymentGroup` as a field. It represents an atomic group of transactions as an array. This is natively supported by Algorand and does not require contracts.
+
+A group can have transactions of different types, such as, for example, `pay` (payment) and `axfer` (asset transfer). A group in Algorand can contain several different types of transactions. For more info, see the [Algorand documentation on transaction types](https://dev.algorand.co/concepts/transactions/reference/).
+
+As part of the payload there **MUST** also be a `paymentTxID` field which identifies the transaction in the group that will pay the resource server. The group may perform several operations to facilitate the payment, such as swaps or asset transfers, but only one transaction in the group will actually transfer the funds to the resource server.
+
+There can be multiple signers in the group and fees can be pooled together or assigned to a specific signer; meaning it can be delegated to a specific account to pay the fees for the group. A max of 16 transactions can be included in a group.
+
+Example:
+
+```json
+{
+  "paymentTxID": "R4PFEBJGLXV4PDOIBLY7UKBLVLXFCPYLXGBILC22BAVILTKWV3TA", // ID of the transaction in the group that wll pay the resource server
+  "paymentGroup": [
+    {
+      "signature": "wVGxbZwPd2c8zKX4RSE4muFRBOHe/nRq5VT6iMeMksi2agfOCTs+VzOGUPJ+bTT5T21EuYjoBKKDX2+/NZG8Cg==",
+      "txn": {
+        "gen": "mainnet-v1.0",
+        "grp": "abcd1234efgh5678ijkl9012mnop3456qrst7890uvwx1234yzab5678cdef9012",
+        "gh": "SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=",
+        "asnd": "PPNHVQJGE6ZMEWPR35UHLUYMCCZV6VNTHLJMZDVCONXKUPV47K4WGQJWGI",
+        "arcv": "ACTSMNPJPS5JMZJJ5G73JOXUUMWXXDGS7TMOER3M4W7BC54ERSZ4KPSMHM",
+        "type": "axfer",
+        "fee": "0",
+        "fv": "6631154",
+        "lv": "6632154",
+        "xaid": "168103"
+      }
+    },
+    {
+      "signature": "6iYMGg8cMw0VYYCfmwru8lixfrVyi1LFXm9H/M0MGf0P4vjK6KGsiCUvsocWUvMo8yTMWQ0hFcP86sRoyGlIAg==",
+      "txn": {
+        "gen": "mainnet-v1.0",
+        "grp": "abcd1234efgh5678ijkl9012mnop3456qrst7890uvwx1234yzab5678cdef9012",
+        "gh": "SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=",
+        "fee": "2000",
+        "fv": "6631154",
+        "lv": "6632154",
+        "snd": "LOXIQKHRCEDEMN5MKBQ32Y54J7H6JKBTEURQL4S65K44MTWN6UMTU2WNBM",
+        "rcv": "LOXIQKHRCEDEMN5MKBQ32Y54J7H6JKBTEURQL4S65K44MTWN6UMTU2WNBM",
+        "amnt": "0",
+        "type": "pay"
+      }
+    }
+  ]
+}
+```
+
```

Comment:
Maybe it's worth adding a note explaining in plain English what this group is doing (e.g., "A payment of `X` USDC with abstracted fee").